Makefile

```
@@ -33,4 +35,4 @@ test:
 	$(GO) test -v ./pkg/...
 
 e2e/test:
-	$(GO) test -v -count=1 ./e2e/...
+	$(GO) test -v -count=$(NUMBER_OF_TIMES) ./e2e/...
```

I am not sure about v1, but github.com/onsi/ginkgo/v2 is not compatible with count flag.
`--count is one of the few go test flags that is not compatible with Ginkgo suites`
https://onsi.github.io/ginkgo/#repeating-spec-runs-and-managing-flaky-specs

```suggestion
	ginkgo -v --repeat=$(NUMBER_OF_TIMES) ./e2e/...
```